app/src/main/java/org/jellyfin/androidtv/integration/LeanbackChannelWorker.kt

```diff
@@ -60,6 +60,7 @@ class LeanbackChannelWorker(
 	 */
 	private val isSupported = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
 		&& context.packageManager.hasSystemFeature("android.software.leanback")
+		&& context.packageManager.resolveContentProvider(TvContractCompat.AUTHORITY, 0) != null
```

Comment:
Perhaps it would be good to add a comment here as to why this check is necessary.

Reply:
Fixed all other comments too (added punctuation and remove line breaks).